config/application.rb

```diff
@@ -40,7 +40,10 @@ class Application < Rails::Application
     end
 
     config.after_initialize do
-      config.action_mailer.notify_settings = { api_key: Rails.application.secrets.notify_api_key }
+      config.action_mailer.notify_settings = {
+        api_key: Rails.application.secrets.notify_api_key,
+        template_id: Rails.application.secrets.notify_template_id,
```

Comment:
I'd just look up the env var here as the template id is not actually a secret.

Reply:
Now changed 👍